src/repositories/donationRepository.ts

```diff
@@ -537,36 +537,17 @@ export async function isVerifiedDonationExistsInQfRound(params: {
   }
 }
 
-export async function findRelevantDonations(
+export async function findDonationsToGiveth(
```

Comment:
I think you can rename this to `findDonationsByProjectId`
because you getting `givethProjectId` as input and actually it could be any project id not only Giveth project

Reply:
I named it this because we have a condition that donation should have useDonationBox = true, so I changed it to findDonationsByIdWitchUseDonationBox which is a better name

Reply:
Thanks @ae2079. There's a typo in `findDonationsByProjectIdWitchUseDonationBox`, please fix (Witch -> which)

Reply:
Done